VelocitySDK/VLTClient.m

```diff
@@ -0,0 +1,211 @@
+//
+//  VLTClient.m
+//  VelocitySDK
+//
+//  Created by Vytautas Galaunia on 17/08/2017.
+//  Copyright © 2017 Veloctity. All rights reserved.
+//
+
+#import "VLTClient.h"
+#import "VLTMotionRecorder.h"
+#import "VLTErrors.h"
+#import "VLTSensorBuilder.h"
+#import "VLTUserDataStore.h"
+#import "VLTCore.h"
+#import "VLTRecordingConfig.h"
+#import "VLTMacros.h"
+#import "VLTProtobufHelper.h"
+#import "VLTConfig.h"
+#import "VLTOperation.h"
+#import "VLTMotionDetectOperation.h"
+#import "VLTCaptureUploadOperation.h"
+
+static void * VLTIsActiveKVOContext = &VLTIsActiveKVOContext;
+
+@interface VLTClient ()
+
+@property (atomic, strong) VLTRecordingConfig *recordingConfig;
+@property (atomic, strong) id <VLTMotionRecorder> recorder;
+@property (atomic, strong) dispatch_source_t hitTimer;
+@property (atomic, assign, getter=isInProgress) BOOL inProgress;
+@property (atomic, assign) UInt32 sequenceIndex;
+
+@property (nonatomic, strong) NSOperationQueue *processingQueue;
+
+@end
+
+@implementation VLTClient
+
+@synthesize active = _active;
```

Comment:
afaik, if you override both - setter & getter compiler throw an error that e.g. `_active` is undeclared identifier.